tests/test_submit_wait_int.py

```diff
@@ -46,7 +44,6 @@ def noexp(job_envs):
 def samdev(job_envs):
     """fixture to run launches for samdev/fermilab"""
     os.environ["GROUP"] = "fermilab"
-    os.environ["EXPERIMENT"] = "samdev"
```

Comment:
Doesn't IFDH need these to work?  Or do we not need them for the tests to pass, but since the job wrappers add `EXPERIMENT`, we're OK?

@marcmengel - do you have any input on deleting these from tests?

Reply:
We do need the EXPERIMENT set in the tests so any SAM stuff can work in the jobs.

Reply:
OK. I will put "EXPERIMENT" back into the tests.